src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/OffenderServiceTest.kt

```diff
@@ -36,19 +36,19 @@ import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas2.OffenderSer
 import java.time.LocalDate
 import java.time.OffsetDateTime
 
-class OffenderServiceTest {
-  private val mockPrisonsApiClient = mockk<PrisonsApiClient>()
+open class OffenderServiceTest {
```

Comment:
Why has this class changed to open and its values to `protected`? I wonder if it's slipped in by mistake. If it's important for fixing the bug, it would be really helpful to document the reasoning in the commit message for future readers (and me!)

Reply:
`PersonUtilsTest` extends `OffenderServiceTest` which requires that `OffenderServiceTest` is declared `open`, also inherited properties are made protected so that they are available to sub-classes only.

Added extra details to commit message and description details above.